docker/test/base/setup_export_logs.sh

```diff
@@ -126,11 +126,14 @@ function setup_logs_replication
     # It's doesn't make sense to try creating tables if SYNC fails
     echo "SYSTEM SYNC DATABASE REPLICA default" | clickhouse-client "${CONNECTION_ARGS[@]}" || return 0
 
+    debug_or_sanitizer_build=$($CLICKHOUSE_CLIENT -q "WITH ((SELECT value FROM system.build_options WHERE name='BUILD_TYPE') AS build, (SELECT value FROM system.build_options WHERE name='CXX_FLAGS') as flags) SELECT build='Debug' OR flags LIKE '%fsanitize%' OR hasThreadFuzzer()")
```

Comment:
Ok, but I'm not sure about the ThreadFuzzer?
Don't we enable it everywhere in the CI?

Reply:
Indeed, fixed, thanks.